### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by gigobyte** [zwilias/elm-xref#6]
The current algorithm for getting elm's home directory expects a [`.elm` folder](https://github.com/zwilias/elm-xref/blob/master/lib/runner.js#L283), but on Windows the folder is under appdata, for example on my machine: 
```
C:\Users\gigobyte\AppData\Roaming\elm\0.19.0\package\NoRedInk\elm-json-decode-pipeline\1.0.0\elm.json
```
instead of
```
C:\Users\gigobyte\.elm\0.19.0\package\NoRedInk\elm-json-decode-pipeline\1.0.0\elm.json
```

This, of course, results in an ENOENT error when trying to run elm-xref.

**Comment by zwilias** [comment]
I've pushed a fix and released it as 4.0.0-beta2 - can you check whether `npx elm-xref@beta` works for you? Once confirmed, I'll release a proper 4.0.0.

Thanks for the report!

**Comment by gigobyte** [comment]
Sorry for taking so long to respond, I couldn't spend a lot of time on my Windows machine this week. I can confirm it indeed works now! Thanks!
